Makefile

```diff
@@ -241,10 +241,13 @@ test:
 	cd tests && cargo test --no-default-features --features "${ENABLE_FEATURES}" ${EXTRA_CARGO_ARGS} -- --nocapture
 
 # This is used for CI test
-ci_test:
+ci_test: doc_test
```

Comment:
Could we run `make test` on CI, rather than having a separate target?

Reply:
The ci_test target was added to speed the CI job, and I think it could cover most of test cases on test target.